host/xtest/regression_4000.c

```diff
@@ -5519,4 +5565,193 @@ static void xtest_tee_test_4014(ADBG_Case_t *c)
 }
 ADBG_CASE_DEFINE(regression, 4014, xtest_tee_test_4014,
 		"Test SM2 KEP (key derivation)");
+
+static void xtest_tee_test_4015(ADBG_Case_t *c)
+{
+	TEEC_Session session = { };
+	uint32_t ret_orig = 0;
+	TEE_OperationHandle op = TEE_HANDLE_NULL;
+	TEE_ObjectHandle keyAllice = TEE_HANDLE_NULL;
```

Comment:
Miss-spelling: `Alice`. Prefer lowercase names for variables: `key_alice`, `key_bob`.

Reply:
> Miss-spelling: `Alice`. Prefer lowercase names for variables: `key_alice`, `key_bob`.

Done!